documentation/modules/adding-hook-using-cli.adoc

```diff
@@ -0,0 +1,130 @@
+kind: Secret
+metadata:
+  name: ssh-credentials
+  namespace: konveyor-forklift
+type: Opaque
+----
+. Encode your playbook by concatenating a file and piping it for base64, for example:
++
+[source,shell]
+----
+$ cat playbook.yml | base64 -w0
+----
++
+[NOTE]
+====
+You can also use a `here` document to encode a playbook:
```

Comment:
could you explain a bit more, as i got a bit confused?

Reply:
@anarnold97 I'm actually wondering if this is still relevant -- it was in the upstream document that I refashioned into the current downstream documentation. I'll make a note to ask Martin and Matthew about it when I ask for their review.

Reply:
I did change the wording.

Reply:
@mnecas Is this note about a here document still relevant?